airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py

```diff
@@ -102,27 +102,36 @@ def github_metadata_definitions(context):
     return Output(metadata_definitions, metadata={"preview": [md.json() for md in metadata_definitions]})
 
 
+def entry_should_be_on_gcs(metadata_entry: LatestMetadataEntry) -> bool:
+    if metadata_entry.metadata_definition.data.supportLevel == "archived":
+        return False
+    if getattr(metadata_entry.metadata_definition.releases, "isReleaseCandidate", False):
+        return False
```

Comment:
Is this accurate? I think the point of truth about whether an entry is still being rolled out (and therefore shouldn't be in the latest bucket) is whether there is an RC in the RC registry.

Reply:
Okay cool, would you mind adding a comment here?